tests/components/neptune_apex/test_config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Test the Neptune Apex config flow."""
+from asynctest import patch
```

Comment:
Please import patch from `tests.async_mock` instead.

Reply:
I don't see async_mock anywhere else in the codebase. The imports for patch that I see are: unittest.mock and asynctest. I have NO knowledge of this place, so please guide me.

Reply:
```py
from tests.async_mock import patch
```

Reply:
Yeah, I think my branch is from before this came in? Should I rebase?

Reply:
Yes, please.